invio/umbrel-app.yml

```
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: invio
+category: finance
+name: Invio
+version: "1.0.0"
+tagline: Self-hosted application platform
+description: >-
+  Invio is a self-hosted application with a modern web interface. It features
+  a frontend web application, providing a complete
+  solution for managing your data and workflows.
+
+  The application includes secure authentication, session management, and
+  persistent data storage. Configure your admin credentials and start using
+  Invio on your Umbrel.
```

We can update the description to be more informative about the project.
```suggestion
description: >-
  Invio is a self-hosted invoicing without the bloat. A fast, minimalist, and
  privacy-friendly invoicing system built for freelancers and small teams who
  want control, not clutter.


  Create and share invoices in seconds, host your data yourself, and share
  secure, password-free links with clients. Lightweight by design, no CRM
  overhead and no SaaS lock-in.
```